lib/cli/src/commands/app/volumes/credentials/s3.rs

```diff
@@ -0,0 +1,120 @@
+//! Retrieve credentials to manage app volumes.
+
+use crate::{
+    commands::{app::util::AppIdentOpts, AsyncCliCommand},
+    config::WasmerEnv,
+};
+
+/// Retrieve access credentials for the volumes of an app.
```

Comment:
File is named `s3.rs` while it should be `credentials.rs`.

Reply:
Ohh it's an extra nested submodule...

Doesn't really make sense to me when the command is called credentials?

Reply:
I thought as follows: `rotate-secrets` is still something related to credentials, but we don't want to have a command hierarchy such as: 
```
wasmer app volumes credentials show
wasmer app volumes credentials rotate-secrets
```
as, imo, it's just too long. Therefore, we wanted to have something like: 
```
wasmer app volumes credentials <...>
wasmer app volumes rotate-secrets
```
Nonetheless, rotating secrets is related to credentials, and that's why there's a nested submodule. The contents of `s3.rs` can go in `mod.rs`, yes; the rationale behind the decision was to expose users to the fact that the infrastructure backing volumes is an s3-compatible storage only when it's needed - but the credentials still are s3 credentials indeed.